packages/packages/documents/src/hooks/use-document-preview-props.ts

```diff
@@ -0,0 +1,14 @@
+import { __ } from '@wordpress/i18n';
+import { EyeIcon } from '../icons/eye-icon';
+import { runCommand } from '@elementor/v1-adapters';
+import useActiveDocument from '../hooks/use-active-document';
+
+export default function useDocumentPreviewProps() {
+	const document = useActiveDocument();
+
+	return {
+		icon: EyeIcon,
+		title: __( 'Preview changes', 'elementor' ),
+		onClick: () => runCommand( 'editor/documents/preview', { id: document?.id } ),
```

Comment:
maybe don't run it when there is no document?
and add a test for that

Reply:
Done.